docs/BR.md

```diff
@@ -1616,15 +1607,23 @@ f. `nameConstraints` (optional)
 
 [^*]: Non-critical Name Constraints are an exception to RFC 5280 (4.2.1.10), however, they MAY be used until the Name Constraints extension is supported by Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
 
-g. `extkeyUsage` (optional)
+g. `extkeyUsage` (optional/required)
+
+   For Cross Certificates that share a Subject Distinguished Name and Subject Public Key with a Root Certificate operated in accordance with these Requirements, this extension MAY be present. If present, this extension SHOULD NOT be marked critical. This extension MUST only contain usages for which the issuing CA has verified the Cross Certificate is authorized to assert. This extension MAY contain the anyExtendedKeyUsage [RFC5280] usage, if the Root Certificate(s) associated with this Cross Certificate are operated by the same organization as the issuing Root Certificate.
+
+   For all other Subordinate CA Certificates, including Technically Constrained Subordinate CA Certificates:
+
+   This extension MUST be present and SHOULD NOT be marked critical[^**].
 
-   For Subordinate CA Certificates to be Technically constrained in line with section 7.1.5, then either the value `id-kp-serverAuth` [RFC5280] or `id-kp-clientAuth` [RFC5280] or both values MUST be present[^**].
+   For Subordinate CA Certificates that will be used to issue TLS certificates, the value id-kp-serverAuth [RFC5280] MUST be present. The value id-kp-clientAuth [RFC5280] MAY be present. The values id-kp-emailProtection [RFC5280], id-kp-codeSigning [RFC5280], id-kp-timeStamping [RFC5280], id-kp-OCSPSigning [RFC5280], and anyExtendedKeyUsage [RFC5280] MUST NOT be present. Other values SHOULD NOT be present.
 
-   Other values MAY be present.
+   For Subordinate CA Certificates that are not used to issue TLS certificates, then the value id-kp-serverAuth [RFC5280] MUST NOT be present. Other values MAY be present, but SHOULD NOT combine multiple independent usages (e.g. including id-kp-timeStamping [RFC5280] with id-kp-OCSPSigning [RFC5280] or id-kp-codeSigning [RFC5280]).
```

Comment:
I am a bit unclear with the example of `id-kp-OCSPSigning [RFC5280]` which is probably due to my lack of in-depth knowledge on this particular EKU. Shouldn't this EKU be included along with other EKUs to allow the CA to sign delegated OCSP responder Certificates?

Reply:
No. This is an EKU that appears on the delegated signer, not the issuing CA. 

You can see this past discussion on m.d.s.p: https://groups.google.com/d/msg/mozilla.dev.security.policy/XQd3rNF4yOo/bXYjt1mZAwAJ

Reply:
I'm not sure I understand the point you're trying to make.

It's not necessary to include. The situation described was to workaround a particular CA software bug, and there are solutions to avoid that bug that do not require placing the EKU within a Sub-CA, which is not consistent with RFC 6960.

It seems like you're advocating that CAs should be able to ignore the OCSP RFC to work around a CA bug, but I could be misunderstanding?

Reply:
I am not suggesting anything in relation to RFC 6960. My point is that your example has a SHOULD that prevents a CA from using the id-kp-OCSPSigning EKU along with -say- id-kp-codeSigning, and I think this particular EKU example is confusing. CAs should be fine including the id-kp-OCSPSigning EKU along with other EKUs. It's other EKUs that should not be allowed to co-exist with other EKUs (for example id-kp-emailProtection with id-kp-codeSigning). 
I hope this makes it somewhat clearer, at least in terms of intent.

Reply:
That particular EKU example is intentional.

You MUST NOT include that EKU on anything other than a delegated OCSP responder.
A delegated responder MUST NOT be capable of issuing certificates.

There's no legitimate use case for including both, except to work around a bug with a particular CA vendor, except that doing so introduces a security vulnerability, so you can and MUST work around the bug differently.

Reply:
See also the discussion above (e.g. https://github.com/sleevi/cabforum-docs/pull/10#issuecomment-631727791 )